packages/patternfly-4/react-docs/src/components/propsTableTs/propsTableTs.js

```diff
@@ -36,19 +36,19 @@ export const PropsTableTs = ({ name, props }) => (
         <TH>Description</TH>
       </Heading>
       <Body>
-        {props.map(prop => (
-          <Row key={prop.name}>
-            <TD>{prop.name}</TD>
-            <TD>{prop.type.name}</TD>
-            <TD align="center">{prop.flags.isOptional === null && <ExclamationCircleIcon />}</TD>
-            {/* <TD>{Boolean(prop.defaultValue) && prop.defaultValue.value}</TD> */}
-            <TD>{prop.comment && prop.comment.shortText}</TD>
-          </Row>
-        ))}
       </Body>
     </Table>
   </Section>
 );
+// {props.map(prop => (
```

Comment:
We should avoid committing commented-out code, right?

Also, when trying to check the effects of removing this, I went to look at the props table for Avatar and on https://1541-pr-patternfly-react-patternfly.surge.sh/patternfly-4/components/avatar I'm getting a blank page with console errors:

```
TypeError: Cannot read property 'name' of null
    at getEnumValue (propsTable.js:68)
    at propsTable.js:53
    at Array.map (<anonymous>)
    at PropsTable (propsTable.js:49)
    at hg (react-dom.production.min.js:150)
    at Tg (react-dom.production.min.js:178)
    at bi (react-dom.production.min.js:232)
    at ci (react-dom.production.min.js:233)
    at Di (react-dom.production.min.js:249)
    at Yh (react-dom.production.min.js:248)
qh @ react-dom.production.min.js:198
react-dom.production.min.js:248 Uncaught TypeError: Cannot read property 'name' of null
    at getEnumValue (propsTable.js:68)
    at propsTable.js:53
    at Array.map (<anonymous>)
    at PropsTable (propsTable.js:49)
    at hg (react-dom.production.min.js:150)
    at Tg (react-dom.production.min.js:178)
    at bi (react-dom.production.min.js:232)
    at ci (react-dom.production.min.js:233)
    at Di (react-dom.production.min.js:249)
    at Yh (react-dom.production.min.js:248)
```

Reply:
Yeah, that's a problem. @jschuler has found a way around it in #1539. For now I'll find a way to disable more functionality for the time being.

Reply:
Fixed in latest commit by changing a little bit of Gatsby logic. New preview should deploy shortly.

Reply:
New preview doesn't seem to have console errors for Avatar: https://1541-pr-patternfly-react-patternfly.surge.sh/patternfly-4/components/avatar